litellm/model_prices_and_context_window_backup.json

```diff
@@ -603,6 +603,13 @@
         "litellm_provider": "mistral",
         "mode": "chat"
     },
+    "mistral/mistral-medium-latest": {
+        "max_tokens": 8192,
+        "input_cost_per_token": 0.00000273,
```

Comment:
should be `27` and `81` based on this https://docs.mistral.ai/platform/pricing/

Reply:
I apologise for the mistake, I just did a quick api call to `https://api.mistral.ai/v1/models` and saw that `mistral-medium-latest` / `mistral-medium` look the same and I assumed they cost the same, will correct this after work.
Also, where did you find the pricing for `mistral/mistral-medium` 
```
        "input_cost_per_token": 0.00000273,
        "output_cost_per_token": 0.00000820,
````
I don't see it anywhere in the docs, wondering if that is still correct